### ISSUE TITLE
FR - custom time period for include_history

### ISSUE THREAD

**Issue by mrmantas** [gadgetchnnel/lovelace-home-feed-card#32]
This is a great card for showing security events, great work!

One feature that would make this much better for my use case would be to customize this value to have an option to specify number_of_days to query the history for, instead of the last day:

> include_history (optional, defaults to false)
> This allows the history of an entity to be displayed, rather than just its current state (states of "unknown" are automatically filtered out). This currently uses the history for the last day.

Maybe that's already in the works?

**Comment by gadgetchnnel** [comment]
I have added a new **history_days_back** option in the latest beta (0.4.0b3). This is the number of days from the start of the current day, and defaults to zero. This is used like this:

```
type: custom:home-feed-card
history_days_back: 7
entities:
  - entity: sensor.my_sensor
    include_history: true
```

**Comment by gadgetchnnel** [comment]
This is now included in the latest non-beta-release, 0.4.0, so closing this.
